src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/controller/TierCalculationController.kt

```diff
@@ -31,8 +30,9 @@ class TierCalculationController(private val tierCalculationService: TierCalculat
   )
 
   @GetMapping("crn/{crn}/tier")
-  fun getLatestTierCalculation(@PathVariable(required = true) crn: String): ResponseEntity<TierDto> {
-    return ResponseEntity.ok(tierCalculationService.getLatestTierByCrn(crn))
+  fun getLatestTierCalculation(@PathVariable(required = true) crn: String): ResponseEntity<String> {
```

Comment:
This doesn't look related to the rest of the PR, but if it is supposed to be here then we could do with a test for it

Reply:
I accidently checked this in. I'll revert.